tsinfer/formats.py

```diff
@@ -703,7 +694,6 @@ def __str__(self):
             ("format_name", self.format_name),
             ("format_version", self.format_version),
             ("finalised", self.finalised),
-            ("uuid", self.uuid),
```

Comment:
We should document this in the CHANGELOG I guess, but totally fine to day "this is gone, deal with it"